docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -190,6 +190,7 @@ Experimental features might be changed or removed without prior notice.
 | `databaseReadReplica`                       | Use a read replica for some database queries.                                                                                                                                                                                                                                     |
 | `alertingApiServer`                         | Register Alerting APIs with the K8s API server                                                                                                                                                                                                                                    |
 | `dashboardRestoreUI`                        | Enables the frontend to be able to restore a recently deleted dashboard                                                                                                                                                                                                           |
+| `appSidecar`                                | Enable the app sidecar feature that allows rendering 2 apps at the same time                                                                                                                                                                                                      |
```

Comment:
When saying "apps" you don't mean app plugins, right? Maybe this is just me, but isn't this feature about enabling to open an app plugin in a side-view? For me at least something like this would explain core of it better.

Reply:
So in more generic terms, you could open a trace app + logs app. That is why I am talking about opening 2 apps at the same time. Also in that case I sort of consider for example dashboard as an App even though it's not implemented as an app plugin yet (but I think we are going there eventually) and would like to be able to have logs app + dashboard as a possible combination at some moment. But yeah if opening an app in a sideview makes more sense here I can change it I don't have strong feelings about it.

Reply:
Thanks for the explanation Andrej! Not crucial, but based on our discussion last week I think we would probably find something like "opening an app in a sideview" more understandable for the first sight.